sdk/formrecognizer/azure-ai-formrecognizer/README.md

```diff
@@ -419,6 +409,17 @@ with Form Recognizer and require Python 3.5 or later.
 * Manage custom models: [sample_manage_custom_models.py][sample_manage_custom_models] ([async_version][sample_manage_custom_models_async])
 * Copy a model between Form Recognizer resources: [sample_copy_model.py][sample_copy_model] ([async_version][sample_copy_model_async])
 
+### Async APIs
+This library also includes a complete async API supported on Python 3.5+. To use it, you must
+first install an async transport, such as [aiohttp](https://pypi.org/project/aiohttp/). Async clients
+are found under the `azure.ai.formrecognizer.aio` namespace.
+
+### Optional Configuration
```

Comment:
Shouldd this be before Next steps?

Reply:
Cala said to move this under Next Steps. Does Java put it somewhere else?

Reply:
We don't have anything equivalent in Java but just in general the content of `Optional Configuration` seemed like something that could go in the Troubleshooting sextion.

Reply:
Yeah, I think you're right - I moved it under troubleshooting.